tensorflow/core/kernels/mlir_generated/build_defs.bzl

```diff
@@ -284,16 +289,44 @@ def _gen_kernel_library(
       extra_args: Extra arguments to pass to the generator tool.
       test_tags: The tags to pass to the generated test.
       test_size: The "size" argument to pass to the test.
+      jit_i64_indexed_for_large_tensors_types: The input types for which to enable
+                                               JIT compilation of i64-indexed kernels for 
+                                               large inputs.
+      output_jit_i64_indexed_for_large_tensors_types: The output types for which to enable JIT 
+                                                      compilation of i64-indexed kernels for 
+                                                      large inputs.  
     """
 
     enable_cpu = bool(platform == "cpu")
-
-    aot_kernels = zip(types, output_types or types, [False for t in types])
```

Comment:
This line and alike might be what is causing your current issue. 

Before, there are things like `output_types or types` meaning it will use `types` if `output_types` is empty or None. 
Now, you assign `output_types = []` if they are not set, which is different from before.

Reply:
Changed.